src/package_manager/apt.ts

```diff
@@ -39,18 +37,23 @@ const distributionSpecificAptDependencies = {
 		"python-rosdep",
 		// python required for sourcing setup.sh
 		"python",
+		"libc++-dev",
+		"libc++abi-dev",
 	],
 	focal: [
 		// python-rosdep does not exist on Focal, so python3-rosdep is used.
 		// The issue with ros-melodic-desktop-full is also non-applicable.
 		"python3-rosdep",
 		// python required for sourcing setup.sh
 		"python",
+		"libc++-dev",
+		"libc++abi-dev",
 	],
 	jammy: [
 		// python-rosdep does not exist on Jammy, so python3-rosdep is used.
 		// The issue with ros-melodic-desktop-full is also non-applicable.
 		"python3-rosdep",
+		// libc++-dev and libc++abi-dev installs intentionally removed becuase https://github.com/ros-tooling/setup-ros/issues/506
```

Comment:
```suggestion
		// libc++-dev and libc++abi-dev installs intentionally removed because https://github.com/ros-tooling/setup-ros/issues/506
```

Reply:
@orensbruli can you also squash the commits into one after integrating this suggestion and then sign-off the commit? (`git commit --amend --no-edit --signoff`)